app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Show the user details by ID</h1>
```

Comment:
Please use [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABXWI3ZX7ONBHKV37RIJSNSGYX3UIJQ) wireframe for the `user show page`.